xml/ay_bigfile.xml

```diff
@@ -4489,6 +4489,36 @@ config:type="boolean"&gt;true&lt;/diag&gt;</screen>
     <remark role="fixme">Add a short description</remark>
    </para>
 
+   <sect2 xml:id="Software.Selections.base_product">
+    <title>Product Selection</title>
+    <para>
+     Starting with SLE-15 all products are distributed using one media. You have to
+     choose which product you wish to install. You can do it explicitly using.
+     <literal>product</literal> option
+     <example>
+       <title>Explicit product selection</title>
+       <para>
+         &lt;software&gt;&lt;products&gt;&lt;product&gt;SLED15&lt;/product&gt;&lt;/products&gt;&lt;/software&gt;
+       </para>
+     </example>
+    </para>
+    <para>
+      In some special cases it can happen that the media contains just one product. In
```

Comment:
Maybe leave away "some", because it does not add any meaning beyond what "special" does.

Again, should this be "...media contain..." (plural) or "...medium contains..." (singular)?